api/src/migrations/associate-records-with-people.js

```diff
@@ -1,9 +1,12 @@
 const _ = require('lodash');
 const db = require('../db');
 const batch = require('../db-batch');
+const { Contact, Qualifier } = require('@medic/cht-datasource');
+const dataContext = require('../services/data-context');
 
 const getClinic = id => {
-  return db.medic.get(id)
+  return dataContext
+    .bind(Contact.v1.get)(Qualifier.byUuid(id))
     .then(clinic => {
       const contact = clinic.contact;
```

Comment:
One big difference in behavior between the Pouch `get` function and the cht-datasource `get` functions is that the Pouch `get` will throw an error (with `status: 404`) if a doc with the given id is not found. But, the cht-datasource `get` functions will just return `null`.  So, we need to add a null-check here:

```js
if (!clinic) {
    return;
}
```

Then, we can remove the `catch` block below.